openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-Infra-setup/BRUC-create-cspc-stripe-pool/create-cspc-stripe-pool

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -x
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/2-setup/BRUC-create-cspc-stripe-pool/create-cspc-stripe-pool node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
```

Comment:
@w3aman change the directory e2e-openshift into e2e-konvoy

Reply:
Changed!